scripts/android/helper.js

```diff
@@ -32,20 +32,47 @@ function addDependencies(buildGradle) {
 }
 
 /*
- * Add 'google()' to the repository repo list
+ * Add 'google()' and Crashlytics to the repository repo list
  */
 function addRepos(buildGradle) {
   // find the known line to match
   var match = buildGradle.match(/^(\s*)jcenter\(\)/m);
   var whitespace = match[1];
 
   // modify the line to add the necessary repo
-  var googlesMavenRepo = whitespace + 'google() // Google\'s Maven repository from cordova-plugin-firebase';
+  // Crashlytics goes under buildscripts which is the first grouping in the file
   var fabricMavenRepo = whitespace + 'maven { url \'https://maven.fabric.io/public\' } // Fabrics Maven repository from cordova-plugin-firebase'
   var modifiedLine = match[0] + '\n' + googlesMavenRepo + '\n' + fabricMavenRepo;
```

Comment:
`googlesMavenRepo` variable doesn't exist yet.  Move the duplicate declaration out of the if/else sections to above this line.

Reply:
This appears to be why the CI build is failing